template/src/remote/comics.ts

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from '@tanstack/react-query'
+import { getLatestComic } from '@api/comics'
+import { mapComic } from '@api/mappers/comicMappers'
+
+export const useLatestComicQuery = () => {
+  return useQuery({
+    queryKey: ['comic', 'latest'],
```

Comment:
This key - `['comic', 'latest']` - can be used in another place to get to the same data (cached data, error and loading states)